src/api/participantsRouter.ts

```diff
@@ -23,6 +23,8 @@ participantsRouter.post('/', async (req, res) => {
       relate: true,
     });
 
+    const participantTypes = await ParticipantType.query().findByIds(data.types!.map((t) => t.id));
+    sendNewParticipantEmail(data, participantTypes);
```

Comment:
what if the participant creation succeeds, but the email fails ? 
Would the catch block below say Unable to create participant?

Reply:
I've added error catch up in the other PR
https://github.com/IABTechLab/uid2-self-serve-portal/pull/54
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/116539741/a45cee11-02d6-4e54-a6f2-d72195455384)